ocs_ci/ocs/cluster.py

```diff
@@ -590,6 +589,22 @@ def get_throughput_percentage(self):
         logging.info(f"The throughput percentage of the cluster is {throughput_percentage}%")
         return throughput_percentage
 
+    def calc_average_throughput(self, num_of_samples=5):
+        """
+        Calculate the cluster average throughput out of a few samples
+
+        Args:
+            num_of_samples (int): The number of samples to take
+
+        Returns:
+            float: The average cluster throughput
+
+        """
+        throughput_vals = [
+            self.get_cluster_throughput() for _ in range(num_of_samples)
```

Comment:
I see the following issues with the way throughput and throughput percentage is calculated. But they are out of scope for this PR. I'm still mentioning it here so that it is recorded. 
1) I believe constants.THROUGHPUT_LIMIT_OSD of 250 is not the right value even for m5.4xlarge. 
2) The throughput assumes that the environment is AWS with instance type m5.4xlarge, so this function cannot be used for other environments.

Reply:
Hi @keesturam actually, here I'm not calculating the TP based on a constant, but rather based on the dynamic value collected by `ceph status`. The limit is being determined as explained in the below `reach_cluster_load_percentage_in_throughput()`. 
`get_cluster_throughput()` is not the method that uses the 250MB constant. The one that does it is `get_throughput_percentage()`